src/schema/v2/homeView/sections/ExploreBy.ts

```diff
@@ -0,0 +1,61 @@
+import { HomeViewSection } from "."
+import { HomeViewSectionTypeNames } from "../HomeViewSection"
+
+export const ExploreBy: HomeViewSection = {
```

Comment:
We may have more "explore by..." categories in the future, so we should name this `ExploreByCategories`

Reply:
Well it was, and @egdbear asked to change it so you guys have to decide which one should be 😅

Reply:
Gonna merge it as it is now, and can come back if we have more renaming requests to be done here before opening the PR on Eigen

Reply:
Agree with @iskounen 's comment and have suggested and even more explicit naming pattern above